src/EventListener/AssetsInstallCommandListener.php

```diff
@@ -67,13 +69,12 @@ public function copySonataCoreBundleAssets(ConsoleTerminateEvent $event)
             return;
         }
 
-        $output = $event->getOutput();
-
         $this->execute($event->getInput(), $event->getOutput(), $application);
     }
 
     protected function execute(InputInterface $input, OutputInterface $output, Application $application): int
     {
+        \assert($application instanceof FrameworkApplication);
```

Comment:
You can change the typehint of the `execute` method.

Reply:
done